examples/knative/powershell/kn-ps-harbor-slack/function.yaml

```diff
@@ -12,7 +12,7 @@ spec:
         autoscaling.knative.dev/minScale: "1"
     spec:
       containers:
-        - image: us.gcr.io/daisy-284300/veba/kn-ps-harbor-slack:1.1
+        - image: registry.cloud-garage.net/rguske/kn-ps-harbor-slack:1.2 # needs to be updated
```

Comment:
Please change this to `us.gcr.io/daisy-284300/veba/kn-ps-harbor-slack:1.2` and once PR is approved, that's when the container will be built and pushed